### PR TITLE
fix issue with disabled migrate button (#137)

### PR DESCRIPTION
**Pull request by kmaranionjc**
## Issues
* [CUT-4254](https://jumpcloud.atlassian.net/browse/CUT-4254) - CUT-4254-Disabled-Migrate-Button-Bug

## What does this solve?
-Fixes an issue with disabled migrate button when username is inputted before JC API key
-Removed unused Test-Button parameters
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Run the app
2. Select domain profile
3. Input username
4. Check both `Install JCAgent` and `Autobind JC User`
5. Input Connect Key
6. Input API Key 
7. Migrate button should be enabled

[CUT-4254]: https://jumpcloud.atlassian.net/browse/CUT-4254?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ModuleChangelog.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 2.7.6
+
+Release Date: August 21, 2024
+
+#### RELEASE NOTES
+
+This fixes an issue with disabled Migrate Button
```
Comment (jworkmanjc) [inline ModuleChangelog.md:7]:
```suggestion
This release addresses a UI issue with the ADMU where the "Migrate Profile" button could not be selected even though all the required fields were satisfied.
```

File: ModuleChangelog.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+#### Bug Fixes:
+```
+* Fixed an issue with Migrate Button disabled when there is a username is inputted before JC API key
```
Comment (jworkmanjc) [inline ModuleChangelog.md:11]:
```suggestion
* Fixed an issue with ADMU UI "Migrate Profile" button where it remained disabled even though all the required fields were satisfied.
```

File: ModuleChangelog.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+```
 ## 2.7.5
 
 Release Date: August 21, 2024
```
Comment (jworkmanjc) [inline ModuleChangelog.md:15]:
```suggestion
Release Date: August 28, 2024
```

Comment (jworkmanjc) [reply ModuleChangelog.md:15]: We should be checking this in our tech debt cards

**Review by jworkmanjc — CHANGES_REQUESTED** [review]
Just some cleanup, so was the issue that we were calling test-button with a parameter that didn't exist? I guess I'm not understanding why it didn't work before?

File: ModuleChangelog.md (hunk 1 above)
Comment (kmaranionjc) [reply ModuleChangelog.md:7]: The issue was that Test-Button needed an Org ID for the if statement logic to enable the migrate button. Before this change, Test-Button gets called before initializing the OrgID when API key is inputted. Fix was to move the Test-Button after OrgId is initialized.

Comment (jworkmanjc) [reply ModuleChangelog.md:7]: Thank you!

**Review by jworkmanjc — APPROVED** [review]

**Review by gweinjc — APPROVED** [review]

**Pull request merged**